### PR TITLE
module test add catch exception

### PR DESCRIPTION
**Pull request by steverweber**
small helper

### CODE REVIEW

File: salt/modules/test.py [hunk 1]
```diff
@@ -476,6 +476,30 @@ def assertion(assertion):
     assert assertion
 
 
+def try_(module, return_try_exception=False, **kwergs):
```
Comment (thatch45) [inline salt/modules/test.py]: kwergs? Should this be kwargs? :)

Comment (steverweber) [inline salt/modules/test.py]: done.

**Comment by thatch45** [comment]
Thanks!

**Comment by steverweber** [comment]
please backport to 2015.2

**Comment by steverweber** [comment]
jenkins /pass/

**Comment by steverweber** [comment]
reopened as:
https://github.com/saltstack/salt/pull/21211

**Pull request closed without merging**